### PR TITLE
fixing imports for python3

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: pyworkflow/utils/mpi.py [hunk 1]
```diff
@@ -31,8 +31,8 @@
 
 import os
 from time import time, sleep
-from cPickle import dumps, loads
-from process import buildRunCommand, runCommand
+from pickle import dumps, loads
```
Comment (azazellochg) [inline pyworkflow/utils/mpi.py]: in python3 pickle is already optimized

File: pyworkflow/protocol/protocol.py [hunk 2]
```diff
@@ -1982,7 +1982,7 @@ def getPackageCitations(self, bibTexOutput=False):
 
     def citations(self):
         """ Return a citation message to provide some information to users. """
-        citations = self.getCitations().values()
+        citations = list(self.getCitations().values())
```
Comment (azazellochg) [inline pyworkflow/protocol/protocol.py]: This partially fixes #24

Comment (pconesa) [reply pyworkflow/protocol/protocol.py]: Partially? For me it works.

Comment (azazellochg) [reply pyworkflow/protocol/protocol.py]: It opens the window but the articles are just lists not links, also open bibtex doesn't work. But I have to double check

Comment (pconesa) [reply pyworkflow/protocol/protocol.py]: Both cases work for me.

**Pull request merged**